src/components/nav-bar/component.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import NavigationBar from './component';
+
+describe('<NavigationBar />', () => {
+  const component = shallow(<NavigationBar />);
+
+  it('checks if navigation bar exists', () => {
+    expect(component.exists()).toBe(true);
```

Comment:
You can add qa tags to each route and test that the URL for each `NavbarLink` is correct rather than testing the existence of the component only.